crates/loam-cli/src/commands/build/mod.rs

```diff
@@ -125,10 +131,12 @@ impl Cmd {
             ));
             cmd.arg("--crate-type=cdylib");
             cmd.arg("--target=wasm32-unknown-unknown");
-            if self.profile == "release" {
+            let profile = self.profile.clone().unwrap_or("release".to_string());
```

Comment:
You can use `.as_ref` on an option to go from `Option<T>` to `Option<&T>` and in this case you can use `as_deref` would would go `Option<String>` to `Option<&str>` preventing the need to clone and use a string.
Second you should have used `unwrap_or_else` because this way you call `release.to_string` even if the `Option` is a `Some`.

```suggestion
            let profile = self.profile.as_deref().unwrap_or("release");
```